src/components/DesktopNewsletter/index.jsx

```diff
@@ -0,0 +1,14 @@
+
+
+export default function DesktopNewsletter() {
+    return(
+    <div className="grid h-screen bg-primary place-items-center">
+        <div className="relative h-4/6 w-1/2 bg-primary-dark rounded-[2.813rem] right-20 bottom-8"/>
+        <div className="absolute  h-4/6 w-1/2 pl-7 bg-secondary-light rounded-[2.813rem]"> 
```

Comment:
For some reason in my browser (safari) the top box is stuck on the left side of the screen. If it works for you, then try downloading another browser such as Firefox or Chrome and see if it still works.